src/api/handlers/page/delete_page_handler.rs

```diff
@@ -0,0 +1,61 @@
+use std::sync::Arc;
+use serde::Serialize;
+use crate::{ error::Result };
+use axum::{Extension, extract::Path, Json};
+use axum::extract::State;
+use crate::avored_state::AvoRedState;
+use crate::error::Error;
+use crate::models::token_claim_model::LoggedInUser;
+use crate::models::validation_error::{ErrorMessage, ErrorResponse};
+
+pub async fn delete_page_handler(
+    Extension(logged_in_user): Extension<LoggedInUser>,
+    Path(page_id): Path<String>,
+    state: State<Arc<AvoRedState>>
+) -> Result<Json<RemovedPageResponse>> {
```

Comment:
Here as we are returning the response with empty body so this line needs to be changed as :

    ) -> Result<impl IntoResponse> {